src/mapml.js

```diff
@@ -1177,7 +1234,7 @@ M.ContextMenu = L.Handler.extend({
         html = '';
 
     el.innerHTML = html + options.text;
-    el.href = '#';
+    el.role="button";
```

Comment:
The Accessibility Object Model isn't implemented in all browsers (https://wicg.github.io/aom/caniuse.html), perhaps my [comment](https://github.com/Maps4HTML/Web-Map-Custom-Element/pull/124#pullrequestreview-458815592) was misleading, we should do `setAttribute('role', 'button')` instead.